src/config/components/artifacts/mod.rs

```diff
@@ -38,6 +38,7 @@ enum ArtifactSystem {
 pub struct ArtifactsConfig {
     pub auto: bool,
     pub cargo_dist: bool,
+    pub package_specific_releases: bool,
```

Comment:
While this is a good addition, I would change the name, maybe to `match_package_names` (seems more immediately obvious to me what it does that way). Also adding it to the docs reference page would be nice, but no worries, I can also do that.

Reply:
I wasn't sure about the naming either, I'll get those updates made a bit later today. Thanks for the quick response!

Reply:
Just pushed these changes, also added a doc entry but feel free to tweaks as needed!

Reply:
actually ill fix that clippy